sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -309,5 +447,15 @@ public async Task GetTokenForTeamsUserWithWrongUserObjectIdShouldThrow()
             }
             Assert.Fail("An exception should have been thrown.");
         }
+
+        private bool IsTokenExpirationValid(TimeSpan expectedTokenExpiration, DateTimeOffset tokenExpiresAfter)
+        {
+            var tokenTimeSpan = tokenExpiresAfter - DateTimeOffset.UtcNow;
+            var tokenSeconds = tokenTimeSpan.TotalSeconds;
+            var expectedSeconds = expectedTokenExpiration.TotalSeconds;
+            var timeDiff = Math.Abs(expectedSeconds - tokenSeconds);
+            var allowedTimeDiff = expectedSeconds * 0.05;
```

Comment:
I would add the 0.05 as a constant at the top of the file and give it a good name.

Reply:
Also, please check if there isn't a built-in assertion method considering deviation or checking a range.
If such assertion would fail, it'd provide a richer message than just "was false, expected true"

Reply:
Sure, that could also work. Goal: have a clear message with expected and actual values so that it's clear why the assertion failed

Reply:
Done.